.gitpod.yml

```diff
@@ -0,0 +1,11 @@
+# This configuration file was automatically generated by Gitpod.
```

Comment:
Why is this needed?

Reply:
not needed, this was auto generated by gitpod , removed this